cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -663,14 +663,15 @@ where:
      - Number of levels of lineage output to return. Defaults to 10. Determines how far back the provenance
        of the data in the lineage chain is calculated.
    * - ``<collapse>``
-     - An optional set of `collapse` types to collapse the lineage output by. By default, lineage output is
-       not collapsed.
+     - An optional set of ``collapse`` types by which to 
+       :ref:`collapse the lineage output <http-restful-api-metadata-lineage-collapse>. 
```

Comment:
somehow this linking did not work for me. Will check the output of the quick build.